calico_test/tests/st/utils/docker_host.py

```diff
@@ -363,3 +373,12 @@ def get_hostname(self):
 
         command = "docker inspect --format {{.Config.Hostname}} %s" % self.name
         return log_and_run(command)
+
+    def writefile(self, filename, data):
+        """
+        Writes a file on a host (e.g. a yaml file for loading into calicoctl).
+        :param filename: string, the filename to create
+        :param data: string, the data to put inthe file
+        :return: Return code of execute operation.
+        """
+        return self.execute("cat << EOF > %s\n%s" % (filename, data))
```

Comment:
It's weird, but it seems to work.  Maybe docker adds the EOF or something.  Since we're rushed I think it's fine as is.